### DISCUSSION TITLE
Projects using scittle

### DISCUSSION THREAD

**Discussion by borkdude** [borkdude/scittle#41] [Show and tell]
Projects using scittle:

- https://bitemporal-visualizer.github.io/

**Comment by kangaroolab** [comment]
I created a note app which offers usage of Clojurescript based on scittle. It has been my devpad which enables me to make note and do development in browser with clojurescript. Welcome everyone to check it out and let me know if any questions/suggestions.

https://kangaroolab.github.io/tippy/

**Reply by borkdude** [reply]
Nice!
